pkg/generators/forwarding/fluentd/fluent_conf_test.go

```diff
@@ -1023,6 +1025,10 @@ var _ = Describe("Generating fluentd config", func() {
 			</label>
 			# Ship logs to specific outputs
 			<label @SECUREFORWARD_RECEIVER>
+				<filter **>
```

Comment:
Why removing labels from forward receivers?

Reply:
because earlier the labels were removed earlier in the fluent pipeline, and thus removed from all the outputs